operator/main.py

```diff
@@ -315,16 +314,17 @@ def deploy_storage_class():
     logging.info(logf("Deployed StorageClass", manifest=filename))
 
 def send_analytics_tracker(ga_id):
-    reqheader = { 'user-agent': 'Kadalu-App', 'accept': '*/*' }
+    reqheader = { 'user-agent': 'Mozilla/5.0 (Kadalu-App)', 'accept': '*/*' }
     url = "https://www.google-analytics.com/collect?v=1&t=pageview"
-    epoch = time.time()
     track_page = "http://kadalu.org/kadalu-operator"
```

Comment:
We can also add version here. (Note: `KADALU_VERSION` env to be added to server and csi yaml.j2 files.)

```
track_page = "http://kadalu.org/kadalu-%s-%s" % (name, os.environ.get("KADALU_VERSION", "latest"))
```

Reply:
Ack!

Reply:
Done